Utils/ProcessArtifactUtils.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "Win32ErrorCodeException.h"
+#include "StringUtils.h"
+
+class ProcessArtifactUtils
```

Comment:
This class is more close to `Stresser` then to `Utils`
Please move it to `Stresser` project, open Filter (Directory) named: `Artifacts` if there is not one already.
Thank you, you are the best <3